client/src/components/Game/Nav.js

```diff
@@ -113,8 +113,8 @@ const Navbar = props => {
           </Hidden>
           <Hidden smDown>
           <Grid item container className={classes.navEnd}>
-            <Button className={classes.game} variant="outlined">
-              New Game
+            <Button onClick={() => onRestart()} className={classes.game} variant="outlined">
```

Comment:
no need to pass a func to call another func,
`onClick={() => onRestart()}` -> `onClick={onRestart}`